src/Apps/IdentityVerification/IdentityVerificationApp/CompleteFailed.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Button, Link, Sans, Serif } from "@artsy/palette"
+import React from "react"
+
+export const CompleteFailed: React.FC = () => {
+  return (
+    <Box px={[2, 3]} mb={6} mt={4}>
+      <Box
+        mx={["auto"]}
+        width={["100%", "80%"]}
+        maxWidth={"400px"}
```

Comment:
```suggestion
        maxWidth="400px"
```

Reply:
When are the `{}` needed? When this is evaluating an expression?

Reply:
Strings work by default, but for other js types including numbers you need `{}`. In some cases they get left behind when a responsive value for palette/styled-system (in the form of an array) gets removed - with the single string they are no longer necessary.

Reply:
We shouldn't be using `{}` unless interpolatred JS is actually being used.